components/uuf-core/src/main/java/org/wso2/carbon/uuf/core/API.java

```diff
@@ -65,7 +65,7 @@ public RequestLookup getRequestLookup() {
      * @param serviceClassName  service class name
      * @param serviceMethodName method name
      * @param args              method arguments
-     * @return
+     * @return result of the method invocation of the best matched OSGi service
```

Comment:
change this to a meaningful text. may be like "returns the invoked OSGi service instance"